### PR TITLE
Add details to readme

### PR DESCRIPTION
**Pull request by smarlowucf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,70 @@
...
+    },
+    "signature": "signature",
+    "pkcs7": "pkcs7",
+    "cloud_provider": "amazon"
+}
+```
+
+This information is pulled from the Amazon Instance metadata endpoint:
+http://169.254.169.254/latest. Note: the exact information in the
+*document* key may vary.
```
Comment (rtamalin) [inline README.md]: just wondering if entry may be a better word than key?
```suggestion
*document* entry may vary.
```

**Review by rtamalin — APPROVED** [review]
Looks good - minor wording suggestion but what is there is still good

File: README.md [hunk 2]
```diff
@@ -1,3 +1,70 @@
...
+## Meter billing
+
+The `meter_billing` function accepts a dictionary mapping of dimension name
+to usage quantity. This information is used to bill the customer for
+the product ID that is configured in the adapter. If there is an exception
+with metered billing the exception is raised.
+
+## Get CSP Name
+
+The `get_csp_name` returns the name of the CSP provider. In this case it
```
Comment (KeithMnemonic) [inline README.md]: The other two descriptions meter_billing and get_account_info  have the word "function" after the quoted name. Should this have it also to be consistent?

**Review by KeithMnemonic — APPROVED** [review]
LGTM

**Pull request merged**